solr/core/src/java/org/apache/solr/handler/admin/ZookeeperInfoHandler.java

```diff
@@ -116,22 +114,22 @@ public Name getPermissionName(AuthorizationContext request) {
   }
 
   /** Enumeration of ways to filter collections on the graph panel. */
-  static enum FilterType {
+  public static enum FilterType {
     none,
     name,
     status
   }
 
   /** Holds state of a single page of collections requested from the cloud panel. */
```

Comment:
[0] This is unrelated to your PR, but the way this is tightly coupled to Solr's Admin UI is a real shame.